ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -533,6 +544,7 @@ objects:
             region:       ${AWS_REGION}
             userTags:
               expirationDate: ${EXPIRATION_DATE}
+            subnets: ${subnets}
```

Comment:
Hmm.  Should the explicit `subnets: []` be an error?  I dunno if it's worth distinguishing between `nil`/unset and explicitly-an-empty-array.  But except for the convenience here, I wouldn't expect folks to set `subnets` at all unless they were using shared subnets.  On the other hand, composing the install-config from a bunch of chunks so we could insert `subnets: ...` if and only if `${subnets}` was not empty seems like a pain, so maybe the convenience is worth it ;).  Thoughts?

Reply:
The install process not-set/empty the same. So it's not worth it.